src/index.html

```diff
@@ -2,16 +2,182 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta 
+    name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        
+        <div class="form-field">
+          <label for="Surname">
+            Surname:
+          </label>
+      
+          <input type="text" name="Surname" 
+            id="Surname" required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="Name">
+            Name:
+          </label>
+          
+          <input type="text" name="Name" 
+          id="Name" required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="Years">
+            How old are You?
+          </label>
+          <input type="number" name="years" 
+          min="1" max="100" 
+          value="12" id="Years"
+          >
+
+        </div>
+
+        <div class="form-field">
+          <label for="DateOfBirth">
+            Full date of birth:
+          </label>
+          <input type="date" name="dateofbirth"
+          id="DateOfBirth"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="term-of-the-aggrement">
+        I accept the term of the agreement
+          </label>
+          <input type="checkbox" name="terms" 
+          id="term-of-the-aggrement" required
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="E-mail">
+            E-mail:
+          </label>
+          <input type="email" name="email" 
+          placeholder="email@example.com" required
+          id="E-mail"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="Password">
+            Password:
+          </label>
+          <input type="password" name="password" 
+          id="Password" required
+          >
+        </div>
+      </fieldset>
+    
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+      Do you love cats?
+          <label>
```

Comment:
```suggestion
        <div class="form-field">
          Do you love cats?
          <label>
```

![image](https://user-images.githubusercontent.com/80700247/226931946-b38f775b-0e12-46e9-acce-7c324d24eda1.png)